### DISCUSSION TITLE
moredoors

### DISCUSSION THREAD

**Discussion by DB4000K** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1160] [Q&A]
I'm just wondering when more 1.12 doors from drillgon's port will be added to 1.7.10, or if maybe I could help in any way. The doors from the 1.12 port are really great for building in this mod.

**Comment by BallOfEnergy1** [comment]
If you're willing to port them yourself then go for it, though I don't think anyone is currently in the process of porting the 1.12.2 doors to the 1.7.10 version.

**Comment by DB4000K** [comment]
where can i find them in the 1.12 port?
